src/channels/CreateDidChannels/types.ts

```diff
@@ -1,12 +1,13 @@
-import { DidUri, KiltAddress } from '@kiltprotocol/sdk-js';
+import type { Did, KiltAddress } from '@kiltprotocol/types';
+
 import { HexString } from '@polkadot/util/types';
 
 import { DAppName } from '../../dApps/AccessChannels/DAppName';
 import { Origin } from '../../dApps/AccessChannels/Origin';
 
 export type CreateDidInput = DAppName & {
   submitter: KiltAddress;
-  pendingDidUri?: DidUri;
+  pendingDidUri?: Did;
```

Comment:
Would changing `pendingDidUrl` to `pendingDid` following the new naming convention of the SDK break anything?